pipeline-scripts/release.groovy

```diff
@@ -452,6 +456,45 @@ def void sendReleaseCompleteMessage(Map release, int advisoryNumber, String advi
     }
 }
 
+def createAdvisoriesFor(ocpVersion) {
+    build(
+        job: "/build/advisories",
+        propagate: false,
+        parameters: [
+            string(name: "VERSION":       value: ocpVersion),
+            string(name: "DATE",          value: determineNextReleaseDate(ocpVersion)),
+            string(name: "ASSIGNED_TO",   value: "openshift-qe-errata@redhat.com"),
+            string(name: "MANAGER",       value: "vlaad@redhat.com"),
+            string(name: "PACKAGE_OWNER", value: "lmeyer@redhat.com"),
+            string(name: "IMPETUS",       value: "standard"),
+            booleanParam(name: "ENABLE_AUTOMATION", value: true),
+            booleanParam(name: "REQUEST_LIVE_IDs",  value: true),
+            booleanParam(name: "DRY_RUN",           value: false),
+        ]
+    )
+}
+
+def determineNextReleaseDate(ocpVersion) {
```

Comment:
niiiice

Reply:
More Java than Groovy, but it reads nice :sweat_smile: